docs/changelog.md

```diff
@@ -4,6 +4,18 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## v2.5.2
+
+- Variables can be used inside nested FOREACH clauses. [#725](https://github.com/memgraph/memgraph/pull/725)
+- FOREACH clause can now use indexes if needed (e.g. in case of MERGE) [#736](https://github.com/memgraph/memgraph/pull/736)
+- C++ API now allows setting and getting node and relationship properties. [#732](https://github.com/memgraph/memgraph/pull/732)
+- OPTIONAL MATCH can now use label property indexes that are referencing the previously matched variables [#736](https://github.com/memgraph/memgraph/pull/736)
```

Comment:
```suggestion
- Variables can be used inside nested FOREACH clauses. [#725](https://github.com/memgraph/memgraph/pull/725)
- FOREACH clause can now use indexes if needed (e.g. in case of MERGE). [#736](https://github.com/memgraph/memgraph/pull/736)
- C++ API now allows setting and getting node and relationship properties. [#732](https://github.com/memgraph/memgraph/pull/732)
- OPTIONAL MATCH can now use label property indexes that are referencing the previously matched variables. [#736](https://github.com/memgraph/memgraph/pull/736)
```